src/spaceone/inventory/service/cloud_service_service.py

```diff
@@ -312,8 +311,9 @@ def list(self, params):
         query = params.get('query', {})
         query = self._append_resource_group_filter(query, params['domain_id'])
         query = self._change_project_group_filter(query, params['domain_id'])
-        query = self._change_tags_filter(query, params['domain_id'])
+        query = self._change_tags_filter(query)
```

Comment:
change function name
`_change_tags_filter` -> `_change_filter_tags`

Reply:
have reflected thank you.